docs/src/user-guide/guides-retention.md

```
@@ -0,0 +1,167 @@
+For example, to configure an archive retention period of 30 days (43,200 minutes):
+```yaml
+archive_output:
+  # Other archive_output settings
+
+  # Retention period for archives, in minutes. 
+  # Set to null to disable automatic deletion.
+  retention_period: 43200
+```
+
+Similarly, to configure a search result retention period of 1 day (1440 minutes):
+```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)